### PR TITLE
'six' package dependency removed

### PR DESCRIPTION
**Pull request by jerinpetergeorge**
# Type of PR (feature, enhancement, bug fix, etc.)
enhancement

## Description
Removed six package dependency and cleaned the codebase.

Fixes https://github.com/django-dbbackup/django-dbbackup/issues/370

## Checklist

- [x] Tests are passing
- [ ] Documentation has been added or amended for this feature / update

### CODE REVIEW

File: dbbackup/tests/test_connectors/test_postgresql.py [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+from mock import patch
 
-from dbbackup.db.postgresql import (
-    PgDumpConnector,
-    PgDumpGisConnector,
-    PgDumpBinaryConnector
-)
 from dbbackup.db.exceptions import DumpError
+from dbbackup.db.postgresql import (PgDumpBinaryConnector, PgDumpConnector,
+                                    PgDumpGisConnector)
```
Comment (jonathan-s) [inline dbbackup/tests/test_connectors/test_postgresql.py:10]: Do you think you could keep the format as before as that is easier to read?

```suggestion
from dbbackup.db.postgresql import (
    PgDumpBinaryConnector,
    PgDumpConnector,
    PgDumpGisConnector
)
```

File: dbbackup/utils.py [hunk 2]
```diff
@@ -1,21 +1,20 @@
 """
 Utility functions for dbbackup.
 """
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```
Comment (jonathan-s) [inline dbbackup/utils.py:5]: Same here. 

```suggestion
from __future__ import (
    absolute_import,
    division,
    print_function,
    unicode_literals
)
```

File: dbbackup/tests/test_utils.py [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
-from dbbackup import utils, settings
-from dbbackup.tests.utils import (ENCRYPTED_FILE, clean_gpg_keys,
-                                  add_private_gpg, COMPRESSED_FILE,
+from dbbackup import settings, utils
+from dbbackup.tests.utils import (COMPRESSED_FILE, DEV_NULL, ENCRYPTED_FILE,
+                                  add_private_gpg, add_public_gpg,
                                   callable_for_filename_template,
-                                  DEV_NULL, add_public_gpg)
+                                  clean_gpg_keys)
```
Comment (jonathan-s) [inline dbbackup/tests/test_utils.py:16]: And change the formatting here as well.

**Comment by jonathan-s** [comment]
Thanks a lot for making these changes!

**Comment by jerinpetergeorge** [comment]
Thanks for the suggestion and will make the changes asap.

BTW, I used the **_isort_** package to sort the imports. IMHO, we should add some kind of config file so that we can keep unified formatting rules across the repository/package. And later, the pipeline will take care of the checks

**Comment by jerinpetergeorge** [comment]
I hope everything is fine now, isn't it?

**Review by jonathan-s — APPROVED** [review]

**Comment by jonathan-s** [comment]
Thanks a lot @jerinpetergeorge

**Pull request merged**